app/models/comfy/cms/page.rb

```diff
@@ -97,7 +97,7 @@ def published_at
     elsif scheduled?
       scheduled_on
     elsif ever_been_published?
-      revisions.detect { |revision| revision.data[:event].include?('published') }.created_at
+      revisions.detect { |revision| Array(revision.data[:event]).include?('published') }.created_at
```

Comment:
I'd rather array-ify it so the next thing knows exactly what API it should use and opposed to a nil guard